helm/idol-eductionserver/README.md.gotmpl

```diff
@@ -0,0 +1,34 @@
+{{ template "chart.header" . }}
+{{ template "chart.deprecationWarning" . }}
+
+{{ template "chart.badgesSection" . }}
+
+{{- $component := "EductionServer" }}
+{{- $appVersion := .AppVersion }}
+{{- $versionList := slice (splitList "." $appVersion) 0 2 }}
+
+{{- $docsLink := printf "https://www.microfocus.com/documentation/idol/IDOL_%s/%s_%s_Documentation/Help/" (join "." $versionList ) $component (join "." $versionList) }}
+
+Provides an IDOL Eduction Server deployment.
+
+Depends on connections to:
+
+- IDOL LicenseServer (`licenseServerHostname`)

```

Comment:
Not sure whether this means we should be including the `idol-licenseserver` subchart as a dependency — @daniel-lau-mf, does this add anything, or is defining the licenseserver hostname sufficient?

Reply:
If the specified `licenseServerHostname` is resolvable by the container then that's sufficient, and doesn't really need a subchart dependency (or it should be optional if it's there).